content/blog/2017-09-04-promcon-2017-recap.md

```diff
@@ -0,0 +1,122 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> was fantastic and I hope to see everyone next year! Not just amazing talks, also awesome attendees and organizers! Thank you!</p>&mdash; Kamila Součková (@AnotherKamila) <a href="https://twitter.com/AnotherKamila/status/898652581353766912">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Not even close to the end of the year but I can already say, <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> was the best and most fun conference of the year!</p>&mdash; Frederic Branczyk (@fredbrancz) <a href="https://twitter.com/fredbrancz/status/898601276895789057">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">We had a blast at <a href="https://twitter.com/hashtag/PromCon?src=hash">#PromCon</a>! Thx <a href="https://twitter.com/juliusvolz">@juliusvolz</a>, Richard Hartman &amp; the <a href="https://twitter.com/hashtag/PromCon?src=hash">#PromCon</a> team for an amazing conference! Auf Wiedersehen! Til next year! <a href="https://t.co/b4DwJdqQcs">pic.twitter.com/b4DwJdqQcs</a></p>&mdash; Weaveworks (@weaveworks) <a href="https://twitter.com/weaveworks/status/898576990797746178">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">pretty stoked from <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> ! it was probably the best conference i&#39;ve been to yet! Hope work more with the community in the future =)</p>&mdash; Lukas (@1uk4sh) <a href="https://twitter.com/1uk4sh/status/898616723762532356">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+While there were things going wrong here and there, we think that the event turned out super well overall for a community-organized conference, and we're happy to see that our attendees felt similarly!
+
+## Thanks
+
+PromCon 2017 would not have been possible without the help of its sponsors,
+speakers, attendees, and organizers. Thanks so much to all of you! Our Diamond
+and Venue sponsors deserve a special mention at this point, since they did
+the most to support us and made all the food, drinks, video recordings, and
+swag possible:
+
+<h3>Diamond</h3>
+<div class="sponsor-logos">
+  <a href="https://www.weave.works/"><img src="/assets/blog/2017-09-04/weave_logo.png"/></a>
```

Comment:
These should be in some sane order.

Have you checked the sizes are reasonable?

Reply:
The GitLab logo always takes up a bit more vertical space, but not sure what to do about that. I can try to experiment with the layout here a bit more...

What is a sane order (besides solving layout issues)?

Reply:
Sorting alphabetically is probably easiest.